server/internal/graphql/resolver/meshsync.go

```diff
@@ -66,28 +68,22 @@ func (r *Resolver) resyncCluster(ctx context.Context, provider models.Provider,
 			defer dbHandler.Unlock()
 
 			r.Log.Info("Dropping Meshery Database")
-			err = dbHandler.Migrator().DropTable(
-				&meshsyncmodel.KubernetesKeyValue{},
-				&meshsyncmodel.KubernetesResource{},
-				&meshsyncmodel.KubernetesResourceSpec{},
-				&meshsyncmodel.KubernetesResourceStatus{},
-				&meshsyncmodel.KubernetesResourceObjectMeta{},
-				&models.PerformanceProfile{},
-				&models.MesheryResult{},
-				&models.MesheryPattern{},
-				&models.MesheryFilter{},
-				&models.PatternResource{},
-				&models.MesheryApplication{},
-				&models.UserPreference{},
-				&models.PerformanceTestConfig{},
-				&models.SmiResultWithID{},
-				models.K8sContext{},
-			)
+			tables, err := dbHandler.Migrator().GetTables()
 			if err != nil {
 				r.Log.Error(err)
 				return "", err
 			}
 
+			for _, table := range tables {
+				if table == "events" {
+					continue
+				}
+				if err := dbHandler.Migrator().DropTable(table); err != nil {
+					r.Log.Error(err)
```

Comment:
@VihasMakwana can this be a MeshKit-style error?

https://docs.meshery.io/project/contributing/contributing-error